make/ncurses.mk

```diff
@@ -90,6 +90,7 @@ $(NCURSES_DIR)/.configured: $(DL_DIR)/$(NCURSES_SOURCE) $(NCURSES_PATCHES) make/
 		$(STAGING_LIB_DIR)/libmenu.* \
 		$(STAGING_LIB_DIR)/libncurses.* \
 		$(STAGING_LIB_DIR)/libpanel.* \
+		$(STAGING_LIB_DIR)/libtinfo.* \
```

Comment:
No point in this addition: no reason to clean `$(STAGING_LIB_DIR)/libtinfo.*` twice.